firmware/thruna/shared/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+if("${TARGET}" STREQUAL "test")
+    list(APPEND HDRS_TO_FAKE 
+        "${SHARED_IO_INCLUDE_DIR}/io_time.h"
+        "${SHARED_IO_INCLUDE_DIR}/io_buzzer.h"
+        "${SHARED_IO_INCLUDE_DIR}/io_led.h"
+        "${SHARED_IO_INCLUDE_DIR}/io_rgbLed.h"
+        "${SHARED_IO_INCLUDE_DIR}/io_switch.h"
+    )
```

Comment:
Now that all the test source files are shared, do you think we need this CMake code?

Reply:
and if we need the cmake code, can we move it under shared/tests/CMakeLists.txt and include it conditionally, cause this is confusing

Reply:
Removed!